aea/protocols/generator/base.py

```diff
@@ -1914,13 +1925,18 @@ def _init_str(self) -> str:
 
         return init_str
 
-    def generate_protobuf_only_mode(self, run_protolint: bool = True) -> None:
+    def generate_protobuf_only_mode(
+        self, language: str = "python", run_protolint: bool = True
```

Comment:
Make `python` a constant and reuse throughout.

Reply:
Done